src/yaml.jl

```diff
@@ -61,11 +62,11 @@ function dftramo_run(filename::AbstractString)
 end
 
 """
-    run_ramo(psphere, psphere_sites, r, ramoinput, ramostatus; low_psphere = Vector{Int}(undef, 0))
+    run_ramo(wd, psphere, psphere_sites, r, ramostatus; low_psphere = Vector{Int}(undef, 0))
 
 Runs the basic raMO code (necessary in a separate function for auto_psphere functionality)
 """
-function run_ramo(psphere, psphere_sites, r, ramoinput, ramostatus; low_psphere = Vector{Int}(undef, 0))
+function run_ramo(wd::AbstractString, psphere, psphere_sites, r, ramostatus; low_psphere = Vector{Int}(undef, 0))
```

Comment:
You could use the signature 
```
run_ramo(psphere, psphere_sites, r, ramostatus; low_psphere = Vector{Int}(undef, 0), wd = pwd())
```
placing `wd` in the keyword arguments with it defaulting to `pwd()` - that way it does not need to be provided every time.

Reply:
Made commits to reflect some changes. Usually this function is run in a created directory that's not the original working directory. Perhaps it's a better idea to refactor this option into the `raMOInput` struct?

Reply:
That might be a good idea, especially if the desired working directory can be read from the input file.